python/README.md

```diff
@@ -0,0 +1,20 @@
+# Build Instructions
+
+## REST API
+
+There is currently a dependency to build from source in order to use the REST API. 
+
+1. Follow the instructions [here](https://github.com/mlc-ai/mlc-llm/tree/main/cpp) to build the CLI from source.
+2. Launch the server at [http://127.0.0.1:8000/](http://127.0.0.1:8000/).
+    ```shell
+    cd mlc-llm
+    python python/mlc_chat/server.py
+    ```
+3. Go to [http://127.0.0.1:8000/docs](http://127.0.0.1:8000/docs) to look at the list of supported endpoints, or run the sample client script to see how to send queries.
```

Comment:
I’m wondering which document the `docs` here is referring to.

Reply:
FastAPI actually automatically generates interactive API documentation at this endpoint, see [here](https://fastapi.tiangolo.com/lo/tutorial/first-steps/#interactive-api-docs). It could be useful for anyone wanting an overview of the API.